app/presenters/step_nav_presenter.rb

```diff
@@ -73,15 +73,31 @@ def steps
 
   def edition_links
     {
-      "pages_part_of_step_nav": parsed_edition_links
+      "pages_part_of_step_nav": pages_part_of_step_nav,
+      "pages_related_to_step_nav": pages_related_to_step_nav,
     }
   end
 
-  def parsed_edition_links
-    StepNavPublisher.lookup_content_ids(parsed_base_paths).values
+  def base_paths_pages_part_of_step_nav
+    @base_paths_pages_part_of_step_nav ||=
+      parsed_base_paths - step_nav.navigation_rules.base_paths_part_of_step_nav
```

Comment:
Should this be `parsed_base_paths - step_nav.navigation_rules.base_paths_related_to_step_nav`?